src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-plotly.py

```diff
@@ -0,0 +1,16 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2021 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+from PyInstaller.utils.hooks import collect_data_files
+
+datas = collect_data_files('plotly', include_py_files=True,
+                           includes=['**/*.py', 'package_data/**/*.*'])
```

Comment:
Do we really need `include_py_files=True` and explicit collection of `.py` files?

I see from plotly/plotly.py#3122 that this is probably an attempt to work-around the lazy imports, but the proper way to deal with those is to collect problematic submodules into `hiddenimports`, i.e.:
```python
from PyInstaller.utils.hooks import collect_submodules
hiddenimports = collect_submodules('plotly.validators')
```

Reply:
Okay I didn’t know PyInstaller could use that. I’ll test it out this evening and resubmit if it works!